src/injected/ui_util.cpp

```diff
@@ -472,13 +468,13 @@ void UI::update_floor_at(float x, float y, LAYER l)
     if (uid == -1)
         return;
     auto ent = get_entity_ptr(uid);
-    if ((ent->type->search_flags & 0x100) == 0 || !test_flag(ent->flags, 3))
+    if (!(ent->type->search_flags & ENTITY_MASK::FLOOR) || !test_flag(ent->flags, 3))
```

Comment:
Is this fine, or should i be using `!= ENTITY_MASK::X` wherever possible?

Reply:
I think it's fine as long as it works, and it's still readable